### PR TITLE
Document the necessary security configuration

### PR DESCRIPTION
**Pull request by JulianKniephoff**

### CODE REVIEW

File: documentation/opencast-installation.md [hunk 1]
```diff
@@ -133,6 +133,47 @@ Additionally you must set the label for the annotation publication channel. Add
...
+
+What you need is something to the effect of the following two lines:
+
+    <sec:intercept-url pattern="/annotation-tool/**" access="ROLE_ANONYMOUS" />
+    <sec:intercept-url pattern="/extended-annotations/**" access="ROLE_ANONYMOUS" />
+
+Note that this grants **everyone** access to every resource
+under the paths `/annotation-tool` and `/extended-annotations`.
+If you don't do anything out of the ordinary this should be fine,
+since all of the Annotation Tool endpoints that need to
```
Comment (mtneug) [inline documentation/opencast-installation.md]: Probably without `that`?

File: documentation/opencast-installation.md [hunk 2]
```diff
@@ -133,6 +133,47 @@ Additionally you must set the label for the annotation publication channel. Add
...
+In a single-tenant setup,
+this can be found at `<opencast-etc>/security/mh_default_org.xml`.
+If you have a multi-tenant setup,
+you probably know where this configuration needs to go
+better than I do. :wink:
+
+What you need is something to the effect of the following two lines:
+
+    <sec:intercept-url pattern="/annotation-tool/**" access="ROLE_ANONYMOUS" />
+    <sec:intercept-url pattern="/extended-annotations/**" access="ROLE_ANONYMOUS" />
```
Comment (mtneug) [inline documentation/opencast-installation.md]: We still have this rule:

```xml
<sec:intercept-url pattern="/extended-annotations/reset" access="ROLE_ADMIN" />
```

I have not followed the development closely. Is the `reset` endpoint still in existence?

File: documentation/opencast-installation.md (hunk 1 above)
Comment (JulianKniephoff) [reply documentation/opencast-installation.md]: Nono, it's a relative clause: "Alle Endpunkte, die müssen, kümmern sich selbst um Authentifizierung."

File: documentation/opencast-installation.md (hunk 2 above)
Comment (JulianKniephoff) [reply documentation/opencast-installation.md]: Nope, it's gone: #210

File: documentation/opencast-installation.md (hunk 1 above)
Comment (rrolf) [reply documentation/opencast-installation.md]: I would say Julian is right. And by miss-using  DeepL it confirms the "that"

**Pull request merged**